src/diagramDrawing.ts

```diff
@@ -1,6 +1,29 @@
 import { appendFileSync, createWriteStream, writeFileSync } from 'fs';
 import { generate } from 'node-plantuml';
 import { OUTPUT_DIR } from '.';
+import { isArrayOfStrings } from './util';
```

Comment:
maybe other functions are also suited to be moved to this file
* objectFromEntries
* trim
?

Reply:
done